Sources/SwiftSyntax/SourceLocation.swift

```diff
@@ -143,15 +140,10 @@ fileprivate struct SourceLocationDirectiveArguments {
   let line: Int
 
   init(_ args: PoundSourceLocationArgsSyntax) throws {
-    guard args.fileName.segments.count == 1,
-      case .stringSegment(let segment) = args.fileName.segments.first!
-    else {
-      throw Error.stringInterpolationInFileName(args.fileName)
-    }
-    self.file = segment.content.text
```

Comment:
I agree that multi-line string literals don’t make any sense here. It’s on my personal list of things that we should consider changing for Swift 6 since this would technically be a source-breaking change.

Reply:
Sorry for the slow reply. I just created a [Source Breaking](https://github.com/apple/swift-syntax/labels/Source%20Breaking) label that we can attach to bugs whose fix is source breaking.

Reply:
Added in https://github.com/apple/swift-syntax/issues/2002